src/backend/cuda/subconfigure.m4

```diff
@@ -206,7 +206,7 @@ if test "${have_cuda}" = "yes" ; then
                     AC_MSG_RESULT([yes])
                 ],
                 [
-                    with_cuda_sm=${$with_cuda_sm/auto/all}
+                    with_cuda_sm=${$with_cuda_sm/auto/all-major}
```

Comment:
I'm still confused by this setting. What do the slashes do?

Reply:
Sorry, forgot to fix that in last push. Just pushed a new version